src/shapes/Text.ts

```diff
@@ -303,11 +315,13 @@ export class Text extends Shape<TextConfig> {
           lineTranslateX += this.measureSize(letter).width + letterSpacing;
         }
       } else {
+        context.trySetLetterSpacing(letterSpacing);
         this._partialTextX = lineTranslateX;
         this._partialTextY = translateY + lineTranslateY;
         this._partialText = text;
 
         context.fillStrokeShape(this);
+        context.trySetLetterSpacing(0);
```

Comment:
I don't think this like is useful. Konva should already reset any changes of the context with `save()` and `restore()`